media/css/mozorg/mpl-2-0.scss

```diff
@@ -5,54 +5,57 @@
 body {
     font-family: serif;
     -moz-hyphens: auto;
+    hyphens: auto;
     font-size: 110%;
     margin-left: auto;
     margin-right: auto;
     max-width: 30em;
     text-align: justify;
 }
 
-h1, h2, h3 {
+h1,
+h2,
+h3 {
     font-variant: small-caps;
 }
 
 h1 {
     -moz-hyphens: none;
+    hyphens: none;
     font-size: 160%;
     text-align: center;
 }
 
 h2 {
     -moz-hyphens: none;
+    hyphens: none;
     font-size: 140%;
     text-align: left;
 }
 
 h3 {
     -moz-hyphens: none;
+    hyphens: none;
     font-size: 120%;
     text-align: left;
 }
 
 blockquote {
-    -moz-hyphens: none;
```

Comment:
Above style blocks all use `-moz-hyphens` and `hyphens`. Should this be the same?

Reply:
Ah, I missed restoring this one after I updated the rule to allow prefixes. Will fix, thanks!